config/config.json

```diff
@@ -1,10 +1,11 @@
 {
-  "GCLOUD_PROJECT": "google.com:gulliver",
-  "CLOUD_BUCKET": "gulliver.google.com.a.appspot.com",
-  "CLIENT_ID": "605287872172-js6omne47i1k79hnfo7d4bdu9rlemslr.apps.googleusercontent.com",
-  "CLIENT_SECRET": "your client secret",
+  "//": "See README.md for more information about what to put here",
+  "GCLOUD_PROJECT": "run `gcloud config get-value project`",
+  "CLOUD_BUCKET": "see https://console.cloud.google.com/storage/browser?project=$GCLOUD_PROJECT",
+  "CLIENT_ID": "see https://console.cloud.google.com/apis/credentials?project=$GCLOUD_PROJECT",
```

Comment:
@hostirosti Is there a way to extract these values using the `gcloud` tool, instead of directing users to a URL for them to extract "manually"? (i.e. for a given project, return associated cloud buckets, client_ids, etc.) I suppose some sort of script could do it, but I don't want to go overboard with automation...